packages/cloud_functions/cloud_functions/lib/src/https_callable_result.dart

```diff
@@ -13,7 +11,7 @@ class HttpsCallableResult<T> {
   final dynamic _data;
 
   /// Returns the data that was returned from the Callable HTTPS trigger.
-  T get data {
-    return _data as T;
+  T? get data {
```

Comment:
Not in the scope of this PR but shouldn't `_data` be typed as `T?` directly instead?

Reply:
Also, why `T?` and not `T`?

Reply:
Yeah I think this should just be `T` @Salakar - same thing we had on `runTransaction` on Firestore.

Reply:
Done